admin/class-admin-help-button.php

```diff
@@ -39,7 +39,7 @@ public function __construct( $url, $screen_reader_text ) {
 	public function __toString() {
 
 		return sprintf(
-			' <a class="yoast-field-group__help" target="_blank" href="%1$s"><span class="yoast-field-group__help-icon"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 12" width="16" height="16" role="img" aria-hidden="true"
+			' <a class="yoast-help" target="_blank" href="%1$s"><span class="yoast-help__icon"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 12" role="img" aria-hidden="true"
```

Comment:
All links that use `target="_blank"` need to inform screen reader users the link will open in a new browser tab. For this purpose, there's `WPSEO_Admin_Utils::get_new_tab_message()` which returns an appropriate visually hidden text.

Reply:
I noticed that function adds a `<span class="screen-reader-text">`. That's included in this piece of code too, is that acceptable too?

Reply:
Yes. Usually, I concatenate `WPSEO_Admin_Utils::get_new_tab_message()` with the closing `</a>'`. E.g.:

`WPSEO_Admin_Utils::get_new_tab_message() . '</a>'`